ui/config/config.js

```diff
@@ -125,11 +125,11 @@ const kDirectoryToCategory = {
     ko: '던전',
   },
   eureka: {
-    en: 'Eureka',
+    en: 'Adventuring Forays',
     de: 'Eureka',
     fr: 'Eurêka',
```

Comment:
```suggestion
    en: 'Adventuring Forays',
    de: 'Eureka', // FIXME
    fr: 'Eurêka', // FIXME
```

Can you mark these as needing translation still? #3165 will let the find translations script find FIXME.